examples/qtsupport/chatwidget.cpp

```diff
@@ -71,7 +71,8 @@ void ChatWidget::init(actor_system& system, const std::string& name,
 }
 
 void ChatWidget::sendChatMessage() {
-  auto cleanup = detail::make_scope_guard([=] { input()->setText(QString()); });
+  auto cleanup
+    = detail::make_scope_guard([=]() noexcept { input()->setText(QString()); });
```

Comment:
This can just move to the end of the function. There is one `return` in the `if-else` tree that we can just remove because it does nothing.